docs/book/how-to/stack-deployment/deploy-a-cloud-stack.md

```diff
@@ -0,0 +1,129 @@
+---
+description: Deploy a cloud stack from scratch with a single click
+---
+
+# Deploy a cloud stack with a single click
+
+In ZenML, the [stack](../../user-guide/production-guide/understand-stacks.md) 
+is a fundamental concept that represents the configuration of your 
+infrastructure. In a normal workflow, creating a stack
+requires you to first deploy the necessary pieces of infrastructure and then 
+define them as stack components in ZenML with proper authentication.
+
+Especially in a remote setting, this process can be challenging and 
+time-consuming, and it may create multi-faceted problems. This is why we 
+implemented a feature, that allows you to **deploy the necessary pieces of 
+infrastructure on your selected cloud provider and get you started on remote 
+stack with a single click**.
+
+{% hint style="info" %}
+If you have the required infrastructure pieces already deployed on your cloud, 
+you can also use [the stack wizard to seamlessly register your stack](register-a-cloud-stack.md).
+{% endhint %}
+
+## How to use the 1-click deployment tool?
+
+To access the 1-click deployment tool, you can either use the dashboard or 
+the CLI:
+
+{% tabs %}
+{% tab title="Dashboard" %}
+
+In order to create a remote stack over the dashboard go to the stacks page 
+on the dashboard and click "+ New Stack".
+
+![The new stacks page](../../.gitbook/assets/register_stack_button.png)
+
+Since we will be deploying it from scratch, select "New Infrastructure" on the
+next page:
+
+![Options for registering a stack](../../.gitbook/assets/register_stack_page.png)
+
+{% hint style="warning" %}
+Currently, the 1-click deployment only works on AWS. We are working on 
```

Comment:
might make sense to indicate that for other clouds you can deploy  manually

Reply:
done.